docs/troubleshooting.md

```
@@ -17,12 +17,54 @@
+
+   # Compare webpack vs rspack configs
+   bin/export-bundler-config --bundler=webpack --output=webpack.txt
+   bin/export-bundler-config --bundler=rspack --output=rspack.txt
+   diff webpack.txt rspack.txt
+   ```
+
+   See `bin/export-bundler-config --help` for all available options.
+
+6. Generate webpack stats for build analysis (useful for bundle size optimization):
+
    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)